### PR TITLE
pwd: add Thai translation

### PR DESCRIPTION
**Pull request by punparin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.th/common/pwd.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# pwd
+
+> แสดงชื่อของไดเรกทอรีที่ทำงานอยู่
+> ดูข้อมูลเพื่มเติม: <https://www.gnu.org/software/coreutils/pwd>.
```
Comment (navarroaxel) [inline pages.th/common/pwd.md:4]: We use this translation
```suggestion
> ข้อมูลเพิ่มเติม: <https://www.gnu.org/software/coreutils/pwd>.
```

**Review by navarroaxel — COMMENTED** [review]
We have 2 `pwd` pages, this translation is in the incorrect directory. https://github.com/tldr-pages/tldr/blob/main/pages/linux/pwd.md and https://github.com/tldr-pages/tldr/blob/main/pages/common/pwd.md

File: pages.th/common/pwd.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# pwd
+
+> แสดงชื่อของไดเรกทอรีที่ทำงานอยู่
+> ข้อมูลเพื่มเติม: <https://www.gnu.org/software/coreutils/pwd>.
```
Comment (navarroaxel) [inline pages.th/common/pwd.md:4]: Why isn't the same string? :thinking: 
```suggestion
> ข้อมูลเพิ่มเติม: <https://www.gnu.org/software/coreutils/pwd>.
```

File: pages.th/linux/pwd.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# pwd
+
+> แสดงชื่อของไดเรกทอรีที่ทำงานอยู่
+> ข้อมูลเพื่มเติม: <https://www.gnu.org/software/coreutils/pwd>.
```
Comment (navarroaxel) [inline pages.th/linux/pwd.md:4]:
```suggestion
> ข้อมูลเพิ่มเติม: <https://www.gnu.org/software/coreutils/pwd>.
```

File: pages.th/common/pwd.md (hunk 2 above)
Comment (punparin) [reply pages.th/common/pwd.md]: Nice catch! It was a typo.

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**